CHANGELOG

```diff
@@ -1,3 +1,8 @@
+Master
+------
+
+* Move SessionStorage from a generated class to a re-usable module
```

Comment:
Is there a migration path from the generated class to using the module? Should we detail it here?